samples/interoperability/qrng/README.md

```diff
@@ -9,7 +9,7 @@ products:
 description: "This sample implements a quantum random number generator using Q#, a good first example to teach how to use the language."
 ---
 
-# Creating random numbers with quantum computing
+# Creating random numbers with quantum computing (host)
```

Comment:
Perhaps this would be a good opportunity to highlight that this is a good sample for understanding interop?

```suggestion
# Run Q# programs from Python or .NET
```